File: Lista 1/Q8.java
import java.util.Scanner;
public class Q8 
{
   public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      System.out.print("Digite o primeiro valor: ");
      int NUM1 = sc.nextInt();
      
      System.out.print("Digite o segundo valor: ");
      int NUM2 = sc.nextInt();
      
      System.out.println("A subtração desses valores resulta em: " + (NUM1-NUM2));
   }
}

File: Lista 3/Q8.c
#include <stdio.h>
    int main(void) {
    int CONT=2, NUM=0, NUM1=0, NUM2=1;
    printf("%d\n", NUM1);
    printf("%d\n", NUM2);
    NUM=NUM1+NUM2;
    while(CONT<=9){
      printf("%d\n", NUM);
    NUM1=NUM2;
    NUM2=NUM;
    NUM=NUM1+NUM2;
    CONT=CONT+1;
}
    
  return 0;
}
